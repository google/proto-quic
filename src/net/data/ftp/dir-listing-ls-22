drwxrwxr-x    5 ftp      ftp         12288 Oct 20 17:04 .
drwxr-xr-x    7 ftp      ftp          4096 Apr 03  2010 ..
-rw-rw-rw-    1 ftp      ftp          4931 Jun 08 15:23 _READ_ME.txt
drwxrwxrwx    4 ftp      ftp          4096 May 31 16:26 Аватар_-_Avatar-(2009)-[1080p]_[BD]
-rwxrwxr-x    1 ftp      ftp      17577705968 Mar 08  2009 Американский_пирог_1_[Расширенная_версия]_-_American_Pie_1_[Unrated_Edition]-(1999)-[1080p]_[BD_remux].ts
-rwxrwxr-x    1 ftp      ftp      15512934868 Mar 16  2009 Большой_куш_-_Snatch-(2000)-[1080i]_[HDTV].ts
drwxrwxrwx    2 ftp      ftp          4096 Jun 03 19:07 Большой_куш_-_Snatch-(2000)-[1080p]_[BD_Remux]
-rwxrwxr-x    1 ftp      ftp      8900589105 Mar 24  2009 Война_миров_-_War_of_the_Worlds-(2005)-[720p]_[HDTV].mkv
-rwxrwxr-x    1 ftp      ftp      27728321654 Mar 09  2009 Гангстер_-_American_Gangster-(2007)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      31731782861 Mar 09  2009 Гангстер_[Расширенная_версия]_-_American_Gangster_[Unrated_Edition]-(2007)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      5009104014 Mar 24  2009 Дорожное_приключение_-_Road_Trip-(2000)-[720p]_[HDTV_Rip].mkv
-rwxrwxr-x    1 ftp      ftp      21410583980 Mar 11  2009 Звёздные_войны-Эпизод_2-Атака_клонов_-_Star_Wars-Episode_2-Attack_of_the_Clones-(2002)-[1080i]_[HDTV].ts
-rwxrwxr-x    1 ftp      ftp      19858181688 Mar 11  2009 Звёздные_войны-Эпизод_3-Месть_Ситхов_-_Star_Wars-Episode_3-Revenge_of_the_Sith-(2005)-[1080i]_[HDTV].ts
-rwxrwxr-x    1 ftp      ftp      29026065728 Mar 16  2009 Звёздный_десант_-_Starship_Troopers-(1997)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      22169179449 Mar 16  2009 Зеркала_[Расширенная_версия]_-_Mirrors_[Unrated_Edition]-(2008)-[1080p]_[BD_remux].mkv
drwxrwxrwx    4 ftp      ftp          4096 Jun 15 14:56 Ниндзя-убийца_-_Ninja_Assassin-(2009)-[1080p]_[BD]
-rwxrwxr-x    1 ftp      ftp      19717173247 Mar 11  2009 Обитель_зла_3_-_Resident_Evil-Extinction-(2007)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      18660904388 Mar 11  2009 Патология_-_Pathology-(2008)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      16476154520 Mar 05  2009 Пила_1_[Режиссёрская_версия]_-_Saw_I_[Director's_Cut]-(2004)-[1080p]_[HDDVD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      19917510515 Mar 05  2009 Пила_2_[Режиссёрская_версия]_-_Saw_II_[Director's_Cut]-(2005)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      18085592265 Mar 05  2009 Пила_3_[Режиссёрская_версия]_-_Saw_III_[Director's_Cut]-(2006)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      3473582701 Mar 05  2009 Пила_4_[Режиссёрская_версия]_-_Saw_IV_[Director's_Cut]-(2007)-[1080p]_[BD_remux].flac
-rwxrwxr-x    1 ftp      ftp      15263958421 Mar 05  2009 Пила_4_[Режиссёрская_версия]_-_Saw_IV_[Director's_Cut]-(2007)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      19944605507 Mar 16  2009 Пила_5_[Режиссёрская_версия]_-_Saw_V_[Director's_Cut]-(2008)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      3024333064 Mar 24  2009 Пингвины_из_Мадагаскара-Операция_С_Новым_Годом!_-_The_Madagascar_Penguins_in_A_Christmas_Caper-(2005)-[1080p]_[BD_remux].ts
-rwxrwxr-x    1 ftp      ftp        125961 Mar 05  2009 Плохой_Санта_[Расширенная_версия]_-_Bad_Santa_[Unrated_Edition]-(2003)-[1080p]_[BD_remux].srt
-rwxrwxr-x    1 ftp      ftp      19908695408 Mar 05  2009 Плохой_Санта_[Расширенная_версия]_-_Bad_Santa_[Unrated_Edition]-(2003)-[1080p]_[BD_remux].ts
-rwxrwxr-x    1 ftp      ftp      23185439267 Mar 11  2009 Побег_из_Шоушенка_-_The_Shawshank_Redemption-(1994)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      19567287274 Mar 16  2009 Тупой_и_еще_тупее_[Расширенная_версия]_-_Dumb_and_Dumber_[Unrated_Edition]-(1994)-[1080p]_[BD_remux].mkv
-rwxrwxr-x    1 ftp      ftp      14773061093 Mar 16  2009 Ураган_-_The_Hurricane-(1999)-[1080p]_[HDDVD_Rip].mkv
-rwxrwxr-x    1 ftp      ftp      22411268500 Mar 11  2009 Хостел_2_[Режиссёрская_версия]_-_Hostel_2_[Director's_Cut]-(2007)-[1080p]_[BD_remux].ts
-rwxrwxr-x    1 ftp      ftp      23712519861 Mar 11  2009 Чужой_против_Хищника_[Расширенная_версия]_-_Alien_vs_Predator_[Unrated_Edition]-(2004)-[1080p]_[BD_remux].mkv
