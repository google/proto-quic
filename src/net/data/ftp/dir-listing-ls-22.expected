d
.
-1
1994
10
20
17
4

d
..
-1
2010
4
3
0
0

-
_READ_ME.txt
4931
1994
6
8
15
23

d
Аватар_-_Avatar-(2009)-[1080p]_[BD]
-1
1994
5
31
16
26

-
Американский_пирог_1_[Расширенная_версия]_-_American_Pie_1_[Unrated_Edition]-(1999)-[1080p]_[BD_remux].ts
17577705968
2009
3
8
0
0

-
Большой_куш_-_Snatch-(2000)-[1080i]_[HDTV].ts
15512934868
2009
3
16
0
0

d
Большой_куш_-_Snatch-(2000)-[1080p]_[BD_Remux]
-1
1994
6
3
19
7

-
Война_миров_-_War_of_the_Worlds-(2005)-[720p]_[HDTV].mkv
8900589105
2009
3
24
0
0

-
Гангстер_-_American_Gangster-(2007)-[1080p]_[BD_remux].mkv
27728321654
2009
3
9
0
0

-
Гангстер_[Расширенная_версия]_-_American_Gangster_[Unrated_Edition]-(2007)-[1080p]_[BD_remux].mkv
31731782861
2009
3
9
0
0

-
Дорожное_приключение_-_Road_Trip-(2000)-[720p]_[HDTV_Rip].mkv
5009104014
2009
3
24
0
0

-
Звёздные_войны-Эпизод_2-Атака_клонов_-_Star_Wars-Episode_2-Attack_of_the_Clones-(2002)-[1080i]_[HDTV].ts
21410583980
2009
3
11
0
0

-
Звёздные_войны-Эпизод_3-Месть_Ситхов_-_Star_Wars-Episode_3-Revenge_of_the_Sith-(2005)-[1080i]_[HDTV].ts
19858181688
2009
3
11
0
0

-
Звёздный_десант_-_Starship_Troopers-(1997)-[1080p]_[BD_remux].mkv
29026065728
2009
3
16
0
0

-
Зеркала_[Расширенная_версия]_-_Mirrors_[Unrated_Edition]-(2008)-[1080p]_[BD_remux].mkv
22169179449
2009
3
16
0
0

d
Ниндзя-убийца_-_Ninja_Assassin-(2009)-[1080p]_[BD]
-1
1994
6
15
14
56

-
Обитель_зла_3_-_Resident_Evil-Extinction-(2007)-[1080p]_[BD_remux].mkv
19717173247
2009
3
11
0
0

-
Патология_-_Pathology-(2008)-[1080p]_[BD_remux].mkv
18660904388
2009
3
11
0
0

-
Пила_1_[Режиссёрская_версия]_-_Saw_I_[Director's_Cut]-(2004)-[1080p]_[HDDVD_remux].mkv
16476154520
2009
3
5
0
0

-
Пила_2_[Режиссёрская_версия]_-_Saw_II_[Director's_Cut]-(2005)-[1080p]_[BD_remux].mkv
19917510515
2009
3
5
0
0

-
Пила_3_[Режиссёрская_версия]_-_Saw_III_[Director's_Cut]-(2006)-[1080p]_[BD_remux].mkv
18085592265
2009
3
5
0
0

-
Пила_4_[Режиссёрская_версия]_-_Saw_IV_[Director's_Cut]-(2007)-[1080p]_[BD_remux].flac
3473582701
2009
3
5
0
0

-
Пила_4_[Режиссёрская_версия]_-_Saw_IV_[Director's_Cut]-(2007)-[1080p]_[BD_remux].mkv
15263958421
2009
3
5
0
0

-
Пила_5_[Режиссёрская_версия]_-_Saw_V_[Director's_Cut]-(2008)-[1080p]_[BD_remux].mkv
19944605507
2009
3
16
0
0

-
Пингвины_из_Мадагаскара-Операция_С_Новым_Годом!_-_The_Madagascar_Penguins_in_A_Christmas_Caper-(2005)-[1080p]_[BD_remux].ts
3024333064
2009
3
24
0
0

-
Плохой_Санта_[Расширенная_версия]_-_Bad_Santa_[Unrated_Edition]-(2003)-[1080p]_[BD_remux].srt
125961
2009
3
5
0
0

-
Плохой_Санта_[Расширенная_версия]_-_Bad_Santa_[Unrated_Edition]-(2003)-[1080p]_[BD_remux].ts
19908695408
2009
3
5
0
0

-
Побег_из_Шоушенка_-_The_Shawshank_Redemption-(1994)-[1080p]_[BD_remux].mkv
23185439267
2009
3
11
0
0

-
Тупой_и_еще_тупее_[Расширенная_версия]_-_Dumb_and_Dumber_[Unrated_Edition]-(1994)-[1080p]_[BD_remux].mkv
19567287274
2009
3
16
0
0

-
Ураган_-_The_Hurricane-(1999)-[1080p]_[HDDVD_Rip].mkv
14773061093
2009
3
16
0
0

-
Хостел_2_[Режиссёрская_версия]_-_Hostel_2_[Director's_Cut]-(2007)-[1080p]_[BD_remux].ts
22411268500
2009
3
11
0
0

-
Чужой_против_Хищника_[Расширенная_версия]_-_Alien_vs_Predator_[Unrated_Edition]-(2004)-[1080p]_[BD_remux].mkv
23712519861
2009
3
11
0
0
